src/browser/provider/built-in/base.js

```diff
@@ -41,7 +41,7 @@ export default class BrowserProviderBase {
         this.resizeCorrections[id] = { width: newWidth - width, height: newHeight - height };
     }
 
-    async resizeWindow (id, pageInfo, width, height) {
+    async resizeWindow (id, width, height, pageSize) {
```

Comment:
Did I get it right: `pageSize` is a size of content window, while `width` and `height` are sizes of content window + borders?